torch/_inductor/compile_fx.py

```diff
@@ -771,9 +669,7 @@ def codegen_and_compile(
                 # In that case, we don't need to run all post compilation steps, we just need
                 # to return the string directly.
                 return compiled_graph
-            compiled_graph = FxGraphCache.post_compile(
-                compiled_graph, example_inputs, cudagraphs, gm
-            )
+            compiled_graph.post_compile2(example_inputs, cudagraphs, gm)
```

Comment:
Why is there post_compile1/post_compile2? It has to do with the FxGraphCache.load path above. I plan to refactor that too, but this is an intermediate step.